web/templates/settings/notification_settings.hbs

```diff
@@ -17,7 +17,7 @@
                         {{> ../help_link_widget link="/help/mobile-notifications#enabling-push-notifications-for-self-hosted-servers" }}
                     </th>
                     <th rowspan="2" width="14%">{{t "Email"}}</th>
-                    <th rowspan="2" width="14%">@all
+                    <th rowspan="2" width="14%"><i class="zulip-icon zulip-icon-at-sign" aria-hidden="true"></i>all
```

Comment:
I don't think we should use `aria-hidden` here. Since we want screen readers to be able to read this. This should have `aria-label` for that.

Reply:
Sure.

Reply:
Changed `aria-hidden:true` to `aria-label:true`.